code/game/objects/items.dm

```diff
@@ -244,6 +244,9 @@
 	if(flags_item & DELONDROP)
 		qdel(src)
 
+///Called whenever an item is unequipped to a new loc (IE, not when the item ends up in the hands)
+/obj/item/proc/dropped_new_loc(mob/user)
+	return
```

Comment:
dropped_from_inventory would be better, no?

Reply:
Up for debates, but is it clear that hands are part of inventory? For me it's confusing

Reply:
What do you mean? 
When EXACTLY is this called